docs/config.json

```diff
@@ -437,6 +437,10 @@
             {
               "title": "teleport-kube-agent",
               "slug": "/kubernetes-access/helm/reference/teleport-kube-agent/"
+            },
+            {
+              "title": "teleport-plugin-jira",
+              "slug": "/kubernetes-access/helm/reference/teleport-plugin-jira/"
```

Comment:
This doesn't really belong in the Kubernetes aAcess section.

Kubernetes access is about using Teleport to access Kubernetes clusters. This helm chart is about deploying Teleport's Jira plugin for access requests to a Kubernetes cluster using Helm.

@ptgott recently made some changes to move some Kubernetes things around, so let's check with him where the most appropriate place to put this would be.

Reply:
Yeah, good point. I've added them here to see if it makes sense, since the chart `teleport-cluster` is also more about running Teleport in Kubernetes rather than accessing a Kube cluster.

Reply:
I think it might be better to move all Helm charts out of this section, but I'm not sure where. @ptgott might have an idea (or a better one than moving).

Reply:
This should go in the `setup/helm-reference` section